connect-button/src/main/java/com/ifttt/connect/ui/AnalyticsObject.java

```diff
@@ -0,0 +1,49 @@
+package com.ifttt.connect.ui;
+
+import com.ifttt.connect.Connection;
+
+class AnalyticsObject {
```

Comment:
`final`

Reply:
cannot be final since `ConnectionAnalyticsObject` extends this class.